desi-extract

```diff
@@ -4,17 +4,26 @@
 DESI spectral extraction code benchmark
 
 salloc -N 5 -t 2:00:00 -C haswell -q interactive
-srun -n 160 -c 2 --cpu_bind=cores ./desi-extract -i $SCRATCH/desi/benchmark/inputs/ -o $SCRATCH/temp
+srun -n 160 -c 2 --cpu_bind=cores ./desi-extract -i $SCRATCH/desi/benchmark/inputs/ -o $SCRATCH/temp $(date +%s)
```

Comment:
Should this be `... --time $(date +%s) ...`? (Looks like it is missing the `--time` part.

Reply:
At some point I fell in line with the POSIX conventions on command line arguments.  That is required arguments don't have `--whatever`, only optional arguments do.  I noticed that you have `required=True` though.  When I saw that I thought we should really just remove the option prefixes altogether since they are required.  But I imagine you have a different standard in DESI (like you force --options to make it more readable or something).

If there were to be a change here I would move to just make it follow POSIX convention actually.

Reply:
oops.  I mis-read the `--time` option that already existed (it is for job time, not for passing in start time).

Indeed for DESI we prefer explicit named arguments even for required args instead of N>>1 unnamed required arguments in a magical order.  In this case I don't care too much as long as the `--help` output tells you what the required positional arguments should be, which does not yet appear to be the case here.